ansible/roles/ansible-horizon-plugin/tasks/pre_tasks.yml

```diff
@@ -2,6 +2,27 @@
 #For Debian: apache2
 #For RHEL: httpd
 
+- name: Find out Horizon path
+  find:
+    paths: "{{ horizon_virtual_env[:17]}}"
```

Comment:
why `:17` ?

Reply:
The default value of __horizon_virtual_env_: '/openstack/venvs/horizon*'_
But we need to select the '/openstack/venvs/' this path to search the horizon dir under virtual environment, hence we are taking this value using _:17_

Reply:
`/openstack/venvs/horizon*` is a pattern that we want to look for, we will proceed when we get 1 result in return.
more like the command `ls /openstack/venvs/horizon*`
In any kind of solution, using indices to truncate a string is considered a bad practice, if at all you want to get the parent directory path, either use `/` to split or use inbuilt directory path modules

Reply:
I agreed. Onward I will keep in my mind will try to avoid such types of fixes.